draft-iyengar-quic-delayed-ack.md

```diff
@@ -125,28 +125,6 @@ endpoint performance in the following ways:
   such links, reducing the number of acknowledgments allows connection
   throughput to scale much further.
 
-Unfortunately, there are undesirable consequences to simply reducing the
```

Comment:
I think deleting this text here leads to the impression that there are only good things that can come out of reducing ACK rate, which leads to the question, why not simply reduce it. I think it's important to note the other points here, and maybe point to the considerations section. How about something like:

> As discussed in {{implementation}}, there are undesirable consequences to a receiver unilaterally
> reducing the acknowledgement frequency however, in terms of congestion control and loss
> detection. Consequently, a sender needs the ability to express its constraints on the acknowledgement 
> frequency to maximize congestion controller performance."